### DISCUSSION TITLE
How to abandon a character?

### DISCUSSION THREAD

**Discussion by jorge-graca-sky** [Lurkars/gloomhavensecretariat#493] [Q&A] [answered]
That is it, how can we do it in the app?

**Comment by jorge-graca-sky** [comment, marked as answer]
ok, the way I found was to go to the top left menu and select remove character.

**Comment by Lurkars** [comment]
I am not sure what you mean with abandon, I see 3 options here
- **remove character** this will simply remove the character, like wipe it from the game as if it never existed afterwards: *Main Menu* -> *Remove Characters*
- **retire character** this will retire the character, add it to the retirement list and apply retirement stuff: on *Character Sheet* check *Personal Quest*, close sheet afterwards and apply
- **absent character** this is only to remove the character for the current/next scenario, so like when a player is missing for a session and you play with 3 instead of 4 characters for example, after finishing/canceling scenario, the character is not absend any more: *Character Menu* (also possible on *Character Sheet*) -> click the *exit icon* next to the character names

**Reply by jorge-graca-sky** [reply]
Abadon is how the rule book describes it: https://pikdonker.github.io/frosthaven-rule-book/#page_65
